test/e2e/providers/aws/aws.go

```diff
@@ -115,30 +115,21 @@ func (a *awsProvider) getInfraID() (string, error) {
 }
 
 // getLatestWindowsAMI returns the imageid of the latest released "Windows Server with Containers" image
-func getLatestWindowsAMI(ec2Client *ec2.EC2, hasCustomVXLANPort bool) (string, error) {
+func getLatestWindowsAMI(ec2Client *ec2.EC2) (string, error) {
 	// Have to create these variables, as the below functions require pointers to them
 	windowsAMIOwner := "amazon"
 	windowsAMIFilterName := "name"
-	windowsAMIFilterValue := ""
-	winDateFilterName := "creation-date"
-	winDateFilterVal := "2022-05-11T*"
+	windowsAMIFilterValue := "Windows_Server-2019-English-Full-ContainersLatest-????.??.??"
```

Comment:
Please move this back below the comments explaining it

Reply:
+1